distribution/ddf-common/src/main/resources/etc/custom.properties

```diff
@@ -385,6 +385,7 @@ org.osgi.framework.bootdelegation = \
     javax.transaction.*, \
     javax.xml.crypto, \
     javax.xml.crypto.*, \
+    jdk.nashorn.*, \
```

Comment:
[KARAF-4603](https://issues.apache.org/jira/browse/KARAF-4603) Nashorn support in Karaf